src/OXRS_LCD.cpp

```diff
@@ -44,18 +47,64 @@ void OXRS_LCD::draw_logo(char * firmware_version)
   strcpy(buffer, "URC v");
   strcat(buffer, firmware_version);
   tft.drawString(buffer, 240/2, 40);
+**/
+  int logo_h = 0;
+  int logo_w = 0;
+  const unsigned char * logo_bm = NULL;
+  uint16_t logo_fg = TFT_BLACK;
+  uint16_t logo_bg = TFT_WHITE;
+  if (strcmp("SHA", fw_maker_code) == 0)
+  {
+    logo_h = sha40Height;
+    logo_w = sha40Width;
+    logo_bm = sha40;
+    logo_fg = tft.color565(0, 165, 179);
+    logo_bg = TFT_WHITE;
+  }
+  if (strcmp("AC", fw_maker_code) == 0)
+  {
+    logo_h = ac40Height;
+    logo_w = ac40Width;
+    logo_bm = ac40;
+    logo_fg = TFT_WHITE;
+    logo_bg = TFT_BROWN;
+  }
+  if (strcmp("BMD", fw_maker_code) == 0)
+  {
+    logo_h = bmd40Height;
+    logo_w = bmd40Width;
+    logo_bm = bmd40;
+    logo_fg = TFT_WHITE;
+    logo_bg = TFT_BLUE;
+  }
+ 
```

Comment:
We should probably have a default logo if the maker code isn't recognised. Did Aaron send you the OXRS icon - that would be perfect as a fallback.

Reply:
Yes we should. That's an open point which will be solved in future. 
As of now I included icons for the 3 known makers in the source, may be I find a (easy) solution that the icon can be sent by the user. Something that needs to be worked out.
 Aaron didn't send anything so far.

Reply:
Can we add that icon Aaron sent?

Reply:
I need to polish it a little before I like to show it

Reply:
Here is the new layout with the (default) OXRS logo and new formatted header text:
![layout_210911](https://user-images.githubusercontent.com/53935853/132951598-59290ec2-ddf3-4c70-a62b-e2ade1ad2072.JPG)

Reply:
Looks great!